docs/mkdocs/docs/roadmap.md

```
@@ -1,16 +1,31 @@
+> be considered an acronym for the Minimum Usable Subset.
 
+- [ ] Run missing renders automatically for `pc render -t readme`
+- [ ] Add search/filter to "Explore" panel.
+- [ ] Add "Steps" in AI modifications, similar how Photoshop deals with history of changes.
+- [ ] Do not spam to all terminals with `partcad` "build" output.
+  - [ ] "$PART_NAME: Showing the part"
+- [ ] Handle Ctrl-C gracefully during build.
+- [ ] Walk through [Tutorial] in clean VM.
+  - There was missing `sudo apt-get install openscad`.
+  - Create Behave test for Tutorial?
+- [ ] Add Quick Start section on https://partcad.org/
```

Markdownlint (0.37.0)

Bare URL used

(MD034, no-bare-urls)